treeple/stats/forest.py

```diff
@@ -86,6 +106,8 @@ def build_coleman_forest(
         Random seed, by default None.
     return_posteriors : bool, optional
         Whether or not to return the posteriors, by default True.
+    use_sparse : bool, optional
+        Whether or not to use a sparse representation for the posteriors.
```

Comment:
Ah good catch this should be documented. Currently they are sparse csc matrices, but actually that behavior will be different than the dense behavior because the dense matrix has nans and the sparse doesn't. I'll change it to return dense matrices that are the same as the dense implementation.